style.css

```diff
@@ -0,0 +1,103 @@
+.navbar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #222;
+  height: 87px;
+  padding: 0px 20px;
+}
+
+.navbar-start {
+  display: flex;
+}
+
+.navbar-end {
+  display: flex;
+}
+
+.navbar-items {
+  display: flex;
+  align-items: center;
+}
+
+.navbar-dropdown {
+  font-family: "Spartan", sans-serif;
+  font-size: 14px;
+  color: #ffffff;
+  text-decoration: none;
+  padding-left: 20px;
+  display: flex;
+  align-items: center;
+}
+
+.main-logo-icon {
+  width: 180px;
+  height: 40px;
+}
+
+.main-logo {
+  margin-right: 20px;
+  display: flex;
+  align-items: center;
+}
+
+.small-icon {
+  width: 18px;
+  height: 18px;
+}
+
+.arrow-container {
+  margin-left: 4px;
+}
+
+.free-trial-button {
```

Comment:
This button is also used elsewhere in the website, so you should name it "navbar-free-trial-button"